### PR TITLE
ES-907669-Modify DocIO GitHub examples repository for document processing playground

### PR DESCRIPTION
**Pull request by Yaminisrisf4389**
Modify DocIO GitHub examples repository for document processing playground

### CODE REVIEW

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Data/Input.docx
Comment (LokeshBaskar) [inline Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Data/Input.docx:1]: @Yaminisrisf4389, why bin folders changes are committed. Please check and update the properly.

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/.gitkeep
Comment (LokeshBaskar) [inline Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/.gitkeep:1]: why bin folders changes are committed. Please check and update the properly.

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/Sample.docx
Comment (LokeshBaskar) [inline Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/Sample.docx:1]: why bin folders changes are committed. Please check and update the properly.

File: Paragraphs/Get-list-value/.NET/Get-list-value/bin/Debug/net8.0/Data/Template.docx
Comment (LokeshBaskar) [inline Paragraphs/Get-list-value/.NET/Get-list-value/bin/Debug/net8.0/Data/Template.docx:1]: why bin folders changes are committed. Please check and update the properly.

File: Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿using Syncfusion.DocIO;
 using Syncfusion.DocIO.DLS;
 
-using (FileStream inputStream = new FileStream(@"../../../Data/Template.docx", FileMode.Open, FileAccess.Read))
+using (FileStream inputStream = new FileStream(@"Data/Template.docx", FileMode.Open, FileAccess.Read))
```
Comment (LokeshBaskar) [inline Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs:4]: Please use the Path.GetFullPath()

File: Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs [hunk 2]
```diff
@@ -14,7 +14,7 @@
             (entity as Break).OwnerParagraph.ChildEntities.Remove(entity);
 
         //Save the Word document
-        using (FileStream outputStream = new FileStream(@"../../../Output.docx", FileMode.Create, FileAccess.Write))
+        using (FileStream outputStream = new FileStream(@"Output/Output.docx", FileMode.Create, FileAccess.Write))
```
Comment (LokeshBaskar) [inline Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs:17]: Please use the Path.GetFullPath()

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]
Please address the feedback

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Data/Input.docx
Comment (Yaminisrisf4389) [reply Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Data/Input.docx:1]: resolved

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/.gitkeep
Comment (Yaminisrisf4389) [reply Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/.gitkeep:1]: resolved

File: Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/Sample.docx
Comment (Yaminisrisf4389) [reply Paragraphs/Apply-paragraph-formatting/.NET/Apply-paragraph-formatting/bin/Debug/net8.0/Output/Sample.docx:1]: resolved

File: Paragraphs/Get-list-value/.NET/Get-list-value/bin/Debug/net8.0/Data/Template.docx
Comment (Yaminisrisf4389) [reply Paragraphs/Get-list-value/.NET/Get-list-value/bin/Debug/net8.0/Data/Template.docx:1]: resolved

File: Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs (hunk 1 above)
Comment (Yaminisrisf4389) [reply Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs]: added

File: Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs (hunk 2 above)
Comment (Yaminisrisf4389) [reply Paragraphs/Remove_all_page_breaks/.NET/Remove_all_page_breaks/Program.cs]: added

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**